### PR TITLE
Update README.md - formatting and freenode test

### PR DESCRIPTION
**Pull request by whaley**
- Fixes README.md indentation on numbered list
- Update README.md with freenode test network steps and updating
configuration in mongo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,34 @@ resolve after running `mvn compile` at least once, then open up the javabot.iml
...
+
+To quickly get javabot running on a live server, ensure you've followed the steps in the Building section and change javabot.properties as follows:
+
+```
+# Connect to freenode test
+javabot.server=testnet.freenode.net 
+javabot.port=9002
+
+# Give your javabot a unique nick
+javabot.nick=tk-421
```
Comment (whaley) [inline README.md]: Inspired by @tgolen

Comment (tgolen) [reply README.md]: Hahaha, I was quite surprised to see this pop-up in my inbox this morning. Well played sir o/

**Comment by whaley** [comment]
What else is worth documenting on README for playing around and manually testing with the bot?  

- Setting up dropwizard for the webapp and using admin?
- Manually editing more collections in mongo?
- Modifying channels to join without /invite?
- Enabling/disabling operations?

**Comment by jottinger** [comment]
I'd think those would work ideally, yeah. I was typing up my list before
reading yours, and it turned out to have the same things on it.

**Comment by whaley** [comment]
Go ahead and merge.  I can open a different PR for the other things we talked about adding to README.md in https://github.com/evanchooly/javabot/pull/209#issuecomment-371204881

**Pull request merged**